src/System.Runtime/tests/Performance/Perf.String.cs

```diff
@@ -91,6 +91,19 @@ public void Contains(int size)
         }
 
         [Benchmark]
+        [MemberData(nameof(ContainsStringComparisonArgs))]
+        public void Contains(StringComparison comparisonType, int size)
+        {
+            PerfUtils utils = new PerfUtils();
+            string testString = utils.CreateString(size);
+            string subString = testString.Substring(testString.Length / 2, testString.Length / 4);
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    for (int i = 0; i < 10000; i++)
+                        testString.Contains(subString, comparisonType);
```

Comment:
The class Perf.String.netcoreapp.cs does not exist yet. If I create it, will it be taken up for testing by naming convention alone?